test/scenarios/rhel-9.2-microshift-source-block-downgrade.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+scenario_create_vms() {
+    prepare_kickstart host1 kickstart.ks.template rhel-9.2-microshift-source-fake-next-minor
```

Comment:
We could combine the next-minor and previous-minor images for a similar test that ensures we block skipping 2 minor versions.

Reply:
This is very good point and I have a feeling that we lost implementation of this when upgrading from 4.13... https://github.com/openshift/microshift/blob/main/pkg/admin/prerun/version.go#L36

Reply:
We could attach a test to https://issues.redhat.com/browse/USHIFT-1088